src/templates/next/components/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, StoryFn } from "@storybook/react"
+import Notification from "./Notification"
+import { NotificationProps } from "~/common/Notification"
+
+export default {
+  title: "Next/Components/Notification",
```

Comment:
issue: I think the site notification should be an internal component, since the user would not use this component directly in the page.

suggestion: move the component into the `shared` folder and use the component directly within the various layouts in the next theme.